### ISSUE TITLE
read the docs search may not be working

### ISSUE THREAD

**Issue by cyrush** [LLNL/conduit#1176]
may need jquery entry in `conf.py`

```
extensions = [
    'sphinx.ext.graphviz',
    'sphinx.ext.autodoc',
    'sphinx.ext.doctest',
    'sphinxcontrib.jquery',
    'sphinx.ext.todo',
    'sphinx.ext.coverage',
    'sphinx.ext.mathjax'
]
```

(path paved by axom)

**Comment by cyrush** [comment]
Seems like a simple fix ---however, adding this may cause problems for spack-based builds, so need to investigate

**Comment by cyrush** [comment]
resolved in #1164
